android/app/src/main/java/com/digginroom/digginroom/feature/room/RoomViewModel.kt

```diff
@@ -56,6 +57,7 @@ class RoomViewModel(
                 rooms.forEachIndexed { index, room ->
                     if (room.roomId == roomId) {
                         rooms[index] = Room(room.videoId, false, room.track, roomId)
```

Comment:
내부적으로는 베리가 작성한 코드와같이 반복문을 돌지만 indexOf를 사용하면 특정 원소의 index를 찾는 코드를 더욱 간소화 시킬 수 있을 것 같슴다~ 
```suggestion
                val index = rooms.indexOf { it.roomId == roomId}
                rooms[Index] = Room(....)
```